src/librustc_codegen_ironox/abi.rs

```diff
@@ -86,7 +86,10 @@ impl ArgTypeMethods<'tcx> for Builder<'a, 'll, 'tcx> {
 pub trait FnTypeExt<'tcx> {
     fn new(cx: &CodegenCx<'_, 'tcx>, sig: ty::FnSig<'tcx>,
            extra_args: &[Ty<'tcx>]) -> Self;
+    /// Return the IronOx `Type` that is equivalent to this type.
     fn ironox_type(&self, cx: &CodegenCx<'a, 'tcx>) -> Type;
+    /// Return the IronOx `Type` that is equivalent to this pointer type.
+    fn ptr_to_ironox_type(&self, cx: &CodegenCx<'a, 'tcx>) -> Type;
```

Comment:
This function's signature seems identical to `ironox_type` which is a bit confusing. I assume it means that it returns the type of (in a sense) `&cx`?